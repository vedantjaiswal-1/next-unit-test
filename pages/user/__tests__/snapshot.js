import { render } from '@testing-library/react'
import UserPage from '../index'

it('renders UserPage unchanged', () => {
  const { container } = render(<UserPage />)
  expect(container).toMatchSnapshot()
})