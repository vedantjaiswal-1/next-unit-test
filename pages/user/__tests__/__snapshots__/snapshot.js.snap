// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders UserPage unchanged 1`] = `
<div>
  <div
    class="container"
  >
    <main
      class="main"
    >
      <h3>
        User List
      </h3>
      <div>
        <div
          class="mb-2 card"
        >
          <div
            class="card-body"
          >
            <p>
              Name: 
              Naruto
               
              Uzumaki
            </p>
            <p>
              Team: 
              7
            </p>
          </div>
        </div>
        <div
          class="mb-2 card"
        >
          <div
            class="card-body"
          >
            <p>
              Name: 
              Sasuke
               
              Uchiha
            </p>
            <p>
              Team: 
              7
            </p>
          </div>
        </div>
        <div
          class="mb-2 card"
        >
          <div
            class="card-body"
          >
            <p>
              Name: 
              Sakura
               
              Haruno
            </p>
            <p>
              Team: 
              5
            </p>
          </div>
        </div>
        <div
          class="mb-2 card"
        >
          <div
            class="card-body"
          >
            <p>
              Name: 
              Kakashi
               
              Hatake
            </p>
            <p>
              Team: 
              1
            </p>
          </div>
        </div>
      </div>
    </main>
  </div>
</div>
`;
